cowdao_cowpy/cow/swap.py

```diff
@@ -141,7 +142,7 @@ async def post_order(
         kind=order.kind,
         partiallyFillable=order.partiallyFillable,
         appData=order.appData,
-        signature=signature.data if signature.data.startswith("0x") else f"0x{signature.data}",
+        signature=signature_data,
```

Comment:
you modified the logic tho. The syntactic sugar check `startswith("0x")` is no longer there

```python3
signature=f"0x{signature_data.removeprefix("0x")}"
 ```
 
 such logic you might want to move the the dataclass `to_string` method tho

Reply:
Yeah actioned already in a later pr, just have the .to_string()